packages/@tinacms/app/src/Playground.tsx

```diff
@@ -73,7 +73,7 @@ const Playground = () => {
         <div className="graphiql-doc-explorer-content">
           <div className="graphiql-markdown-description">
             {noAutoQueries
-              ? 'No autoqueries found, the Tina dev or build command was likely run with the "--noSDK" option. Learn about how to generate autoqeries.'
+              ? 'No autoqueries found, the Tina config is likey set to client.skip = true. Learn about how to generate autoqeries.'
```

Comment:
```suggestion
              ? 'No autoqueries found. The Tina config is likely set to "client.skip = true". Learn about how to generate autoqueries.'
```
^ Not new to this, but kind of seems like we meant to include a link at the end of that. I also don't think we refer to "autoqueries" in our docs.

Reply:
Will follow up with a link in another PR. There are some other playground improvements needed.